### PR TITLE
feat: adding the api examples for payment links

### PR DESCRIPTION
**Pull request by sahil-holacash**

### CODE REVIEW

File: node/payment-links.js [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+    const antifraudMetadataBase64 = Buffer.from(JSON.stringify(antifraudMetadata)).toString('base64');
+    const options = {
+        method: 'GET',
+        headers: {
+            'X-Api-Client-Key': api_key,
+            'X-Cash-Anti-Fraud-Metadata': antifraudMetadataBase64,
+            'Content-Type': 'application/json'
+        },
+    };
+    const url = `https://api-v2.sandbox.holacash.mx/v2/payment_link?limit=1`;
```
Comment (esteban-cash) [inline node/payment-links.js]: @sahil-holacash this is the internal URL, you should use the external public API URL: `https://sandbox.api.holacash.mx/v2`

File: node/payment-links.js [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+    const antifraudMetadataBase64 = Buffer.from(JSON.stringify(antifraudMetadata)).toString('base64');
+    const options = {
+        method: 'POST',
+        headers: {
+            'X-Api-Client-Key': api_key,
+            'X-Cash-Anti-Fraud-Metadata': antifraudMetadataBase64,
+            'Content-Type': 'application/json'
+        },
+    };
+    const url = 'https://api-v2.sandbox.holacash.mx/v2/payment_link';
```
Comment (esteban-cash) [inline node/payment-links.js]: @sahil-holacash this is the internal URL, you should use the external public API URL: https://sandbox.api.holacash.mx/v2

File: node/payment-links.js [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+                }
+                reject(parsedData);
+            });
+        }).on('error', err => reject(err));
+        request.write(JSON.stringify(body));
+        request.end();
+    });
+};
+
+const getPaymentLinkDetail = (payment_link_id, api_key, antifraudMetadata) => {
```
Comment (esteban-cash) [inline node/payment-links.js]: @sahil-holacash call it only `getPaymentLink`.

File: node/payment-links.js [hunk 4]
```diff
@@ -0,0 +1,182 @@
+const https = require('https');
+const Buffer = require('buffer').Buffer;
+
+const getPaymentLinkList = (api_key, antifraudMetadata) => {
```
Comment (esteban-cash) [inline node/payment-links.js:5]: @sahil-holacash add a comment to each method please. Examplaining what it does

File: node/payment-links.js (hunk 1 above)
Comment (sahil-holacash) [reply node/payment-links.js]: updated, FYI @esteban-cash same URL that I used is used in many other files

File: node/payment-links.js (hunk 2 above)
Comment (sahil-holacash) [reply node/payment-links.js]: updated, FYI @esteban-cash same URL that I used is used in many other files

File: node/payment-links.js (hunk 3 above)
Comment (sahil-holacash) [reply node/payment-links.js]: updated

File: node/payment-links.js (hunk 4 above)
Comment (sahil-holacash) [reply node/payment-links.js:5]: updated

**Review by ivan-cash — APPROVED** [review]
I tested in my Postman in sandbox environment with the examples you provided and everything seems ok.

### Create a payment link:
image

### Update payment link:
image

**Pull request merged**